PINRemoteImage.xcodeproj/project.pbxproj

```diff
@@ -964,6 +1009,8 @@
 				PROVISIONING_PROFILE_SPECIFIER = "";
 				SDKROOT = appletvos;
 				SKIP_INSTALL = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
```

Comment:
Is this only for the debug build? We don't want this to be set for release versions I don't think.